app/cypress/e2e/0-ndr-core-tests/privacy_page.cy.js

```diff
@@ -0,0 +1,62 @@
+const { Roles } = require('../../support/roles');
+
+describe('Privacy Page', () => {
+    const baseUrl = Cypress.config('baseUrl');
+    const startUrl = '/';
+    const privacypolicyUrl = '/privacy-policy';
+    const rolesToTest = [Roles.GP_ADMIN, Roles.GP_CLINICAL, Roles.PCSE];
+
+    const loginAndVisitPrivacyPolicyPage = (role) => {
+        cy.login(role);
+        cy.visit(privacypolicyUrl);
+    };
+
+    it('verify privacy policy link in start up page', { tags: 'regression' }, () => {
+        cy.visit(startUrl);
+        cy.url().should('eq', baseUrl + startUrl);
+        cy.get('.nhsuk-footer__list-item-link')
+            .should('exist')
+            .and('have.attr', { target: '_blank' });
+        cy.getByTestId('privacy-link').invoke('removeAttr', 'target').click();
+        cy.url().should('eq', baseUrl + privacypolicyUrl);
+    });
+
+    rolesToTest.forEach((role) => {
```

Comment:
it feels like we would benefit from a slightly different approach here. 

We are adding a decent overhead to the total test execution time by running these tests 3 times (once for each role) even though the test assertions do not change dynamically based on each role.

To me, it makes more sense to expand the `auth_routes` tests to include all routes which should be accessible for each role type (including `/privacy-policy` and then include an individual test for the privacy-policy contents if these are required.

Reply:
discussed with Dev.. the contents are tested as part of unit tests so checking contents are not really required.